nixos/modules/module-list.nix

```diff
@@ -650,7 +650,6 @@
   ./services/networking/fireqos.nix
   ./services/networking/firewall.nix
   ./services/networking/flannel.nix
-  ./services/networking/flashpolicyd.nix
```

Comment:
We'll need release notes changes for the removal of the module.

(And also for every other changes at once as a highlight of the release I guess.)

Reply:
Done!